### PR TITLE
Replace REDIS bug fix

### PR DESCRIPTION
**Pull request by ravivangara17**

### CODE REVIEW

File: components/cookbooks/ring/recipes/replace.rb [hunk 1]
```diff
@@ -1,2 +1,18 @@
...
+deps.each do |dep|
+  class_name = dep['ciClassName'].split('.').last
+  db_type = dep['ciClassName'].split('.').last.downcase
+  if class_name == "Redisio"
+    isRedis = true
+    include_recipe "ring::replace_#{db_type}"
+  end
+end
+
+if !isRedis
```
Comment (schwankl) [inline components/cookbooks/ring/recipes/replace.rb]: ring::add still gets called on line 18

File: components/cookbooks/ring/recipes/replace.rb [hunk 2]
```diff
@@ -1,2 +1,18 @@
+deps = node.workorder.payLoad.DependsOn
+
+db_type = nil
+isRedis = false
+deps.each do |dep|
+  class_name = dep['ciClassName'].split('.').last
+  db_type = dep['ciClassName'].split('.').last.downcase
+  if class_name == "Redisio"
```
Comment (schwankl) [inline components/cookbooks/ring/recipes/replace.rb]: why not use db_type to match on? its needed to generate the recipe name

File: components/cookbooks/ring/recipes/replace.rb (hunk 1 above)
Comment (ravivangara17) [inline components/cookbooks/ring/recipes/replace.rb]: This is a check for REDIS replacement, if it is not redis (might be a cassandra replace) then it calls redis.add.

It works in MAIN-1 circuit

File: components/cookbooks/ring/recipes/replace.rb [hunk 3]
```diff
@@ -1,2 +1,16 @@
+deps = node.workorder.payLoad.DependsOn
 
-include_recipe "ring::add"
+db_type = nil
+is_redis = false
+deps.each do |dep|
+  class_name = dep['ciClassName'].split('.').last
+  db_type = dep['ciClassName'].split('.').last.downcase
+  if class_name == "Redisio"
+    isRedis = true
```
Comment (schwankl) [inline components/cookbooks/ring/recipes/replace.rb]: you didnt change isRedis here.  also, lets use db_type only and get rid of class_name

**Pull request merged**